examples/epd2in13_v2.rs

```diff
@@ -90,32 +89,40 @@ fn main() -> Result<(), std::io::Error> {
     display.clear_buffer(Color::White);
 
     // draw a analog clock
+    let style = PrimitiveStyleBuilder::new()
+        .stroke_color(Black)
+        .stroke_width(1)
+        .build();
+
     let _ = Circle::new(Point::new(64, 64), 40)
-        .into_styled(PrimitiveStyle::with_stroke(Black, 1))
+        .into_styled(style)
         .draw(&mut display);
     let _ = Line::new(Point::new(64, 64), Point::new(30, 40))
-        .into_styled(PrimitiveStyle::with_stroke(Black, 4))
+        .into_styled(style)
```

Comment:
I need to change the style to 4 points of stroke_width

Reply:
```suggestion
        .into_styled(PrimitiveStyle::with_stroke(Black, 4))
```
Doesn't this still work? Which is even useful here since we have a seperate style used only once?

Reply:
I overlooked that convenience method. Much better.

Reply:
Then I think we can use it everywhere where the same style is only used once? Everything else looks good to me :+1: